packages/modular-site/src/App.tsx

```diff
@@ -1,3 +1,56 @@
-export default function App(): string {
-  return 'This is the App';
+// from https://github.com/finos/perspective/tree/master/examples/react
+
+import { useEffect, useRef } from 'react';
+
+import perspective, { Table } from '@finos/perspective';
+import '@finos/perspective-viewer';
+import '@finos/perspective-viewer-datagrid';
+import '@finos/perspective-viewer-d3fc';
+import './index.css';
+import '@finos/perspective-viewer/dist/umd/material.css';
+import type {
+  HTMLPerspectiveViewerElement,
+  PerspectiveViewerOptions,
+} from '@finos/perspective-viewer';
+
+// @ts-ignore
+import superstore from 'superstore-arrow';
+
+import './App.css';
+
+const worker = perspective.shared_worker();
+
+const getTable = async (): Promise<Table> => {
+  const req = fetch(superstore);
+  const resp = await req;
+  const buffer = await resp.arrayBuffer();
+  return worker.table(buffer);
+};
+
+const config: PerspectiveViewerOptions = {
+  'row-pivots': ['State'],
+};
+
+export default function App(): JSX.Element {
+  const viewer = useRef<HTMLPerspectiveViewerElement>(null);
+
+  useEffect(() => {
+    void getTable().then(
+      (table) => {
+        if (viewer.current) {
+          viewer.current.load(table);
+          void viewer.current.restore(config);
+        }
+      },
+      (err) => console.error(err),
+    );
+  }, []);
+
+  // You can also the use the stringified config values as attributes
+  return (
+    //eslint-disable-next-line react/react-in-jsx-scope
```

Comment:
we need to disable this in our eslint config too, this isn't needed anymore with the new jsx runtime

Reply:
Woah funky this is gonna change a lot!

Reply:
Full on! Thankfully there's a codemod, that should be a fun exercise. createElement still works too, so it's backward-compatible.